INSTALL.rst

```diff
@@ -343,6 +343,10 @@ openPMD API
 
   - ``export CMAKE_PREFIX_PATH="$HOME/lib/openPMD-api:$CMAKE_PREFIX_PATH"``
   - ``export LD_LIBRARY_PATH="$HOME/lib/openPMD-api/lib:$LD_LIBRARY_PATH"``
+- If PIConGPU is built with openPMD output enabled, the JSON library
```

Comment:
The indention is wrong.

Reply:
Is it? This is supposed to be a top-level bullet-point, not part of the two above.

Reply:
ahh ok in that case it's fine.